app/register/page.tsx

```diff
@@ -103,10 +105,19 @@ const Register = (): JSX.Element => {
   ): Promise<void> => {
     try {
       await registerAccount(data);
-      await loginAccount(data);
-      router.push('/weeklyPicks');
+      await login(data);
+      router.push('/league/all');
+      toast.custom(
+        <Alert
+          variant={AlertVariants.Success}
+          message="You have successfully registered your account."
```

Comment:
Please sort alphabetically

Reply:
I feel the need to not alphabetize this as I always define the type of variant first before providing a custom message. The message is always based on the variant type.